### PR TITLE
add new comments to categorize properties #197

### PR DESCRIPTION
**Pull request by bttanakagn**
Please review #197.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -201,10 +201,12 @@
     </dependencies>
```
Comment (btyoshikawaa) [inline pom.xml]: - [ ] Is dependency to `selenium` unnecessary ??

File: pom.xml [hunk 2]
```diff
@@ -201,10 +201,12 @@
...
 
     <properties>
-        <java-version>1.8</java-version>
-        <encoding>UTF-8</encoding>
+        <!-- == Maven Plugin Versions == -->
         <org.codehaus.mojo.build-helper-maven-plugin.version>1.9.1</org.codehaus.mojo.build-helper-maven-plugin.version>
         <maven-war-plugin.version>2.5</maven-war-plugin.version>
+        <!-- == Project Properties == -->
+        <java-version>1.8</java-version>
+        <encoding>UTF-8</encoding>
```
Comment (btyoshikawaa) [inline pom.xml]: - [ ] It is better to align the order of the property to `terasoluna-gfw-web-multi-blank `...

File: pom.xml (hunk 1 above)
Comment (bttanakagn) [inline pom.xml]: @btyoshikawaa 
It's necessary. So I added it. e4ccb8db8bcc30fd9f455b696bf13737aa083fef

File: pom.xml (hunk 2 above)
Comment (bttanakagn) [inline pom.xml]: I modified the order. af43dd6ec43bb8f4c8bcef4ecab3166cd86431df

File: pom.xml (hunk 1 above)
Comment (btyoshikawaa) [inline pom.xml]: @bttanakagn 
It is better here.
* `Selenium Test` -> `End to End Test`.
* Or, `Unit Test` -> `Test`, and remove `Selenium Test`.

Comment (bttanakagn) [inline pom.xml]: @btyoshikawaa 
I modified the comment. fc7ea2c4addde3e481ae4e704b0067f828754905

**Pull request merged**